src/app/clusters/thermostat-server/thermostat-server.cpp

```diff
@@ -349,11 +349,11 @@ MatterThermostatClusterServerPreAttributeChangedCallback(const app::ConcreteAttr
         requested = static_cast<int16_t>(chip::Encoding::LittleEndian::Get16(value));
         if (!HeatSupported)
             return imcode::UnsupportedAttribute;
-        if (requested < AbsMinHeatSetpointLimit || requested < MinHeatSetpointLimit || requested > AbsMaxHeatSetpointLimit)
+        if (requested < AbsMinHeatSetpointLimit || requested < MinHeatSetpointLimit || requested < AbsMaxHeatSetpointLimit)
```

Comment:
This change doesn't seem correct. Shouldn't we return `InvalidValue` if the requested value is superior to the `AbsMaxHeatSetpointLimit` ?